charts/backstage/values.yaml

```
@@ -12,8 +12,9 @@ global:
     # Every item defines the plugin `package` as a [NPM package spec](https://docs.npmjs.com/cli/v10/using-npm/package-spec),
     # an optional `pluginConfig` with plugin-specific backstage configuration, and an optional `disabled` flag to disable/enable a plugin
     # listed in `includes` files. It also includes an `integrity` field that is used to verify the plugin package [integrity](https://w3c.github.io/webappsec-subresource-integrity/#integrity-metadata-description).
-    plugins: []
-
+    plugins:
+      - package: "https://npm.registry.redhat.com/@redhat/backstage-plugin-orchestrator-backend-dynamic/-/backstage-plugin-orchestrator-backend-dynamic-1.6.0.tgz"
+        disable: false
```

```suggestion
    plugins: []
```

This list should remain empty by default. The orchestrator plugins are defined in this file: https://github.com/redhat-developer/rhdh-chart/blob/main/charts/backstage/templates/_partials.tpl

---

I have update the plugins list in the values.yaml file to remain empty